packages/block-editor/src/components/list-view/block-select-button.js

```diff
@@ -79,7 +94,23 @@ function ListViewBlockSelectButton(
 				aria-hidden={ true }
 			>
 				<ListViewExpander onClick={ onToggleExpanded } />
-				<BlockIcon icon={ blockInformation?.icon } showColors />
+				{ positionLabel ? (
+					<Tooltip text={ positionLabel }>
```

Comment:
Could the tooltip instead be a child of the whole focusable list view item? I think that will allow it to display when the item is focused too. As it is, it only displays on hover so is not keyboard accessible.

Reply:
Oh, good question. Would that then make the Tooltip too prominent for users moving up and down through the list view? Currently when you're within a list view item, you can press the right arrow to focus on the settings dropdown button, and it might look odd if you're switching between tooltips 🤔 

<img width="416" alt="image" src="https://user-images.githubusercontent.com/14988353/227428156-96595f2d-a7c1-4ad6-bc7c-d884883b13fb.png">

Happy to try it out on Monday, though!

Reply:
Good point, we'd have to try it and see. Assuming the position tooltip would only show when a block is sticky/fixed, it shouldn't be too disruptive?

Reply:
Looking at this again, I'm not sure if we can get the Tooltip on the root element here (so, displaying when the button is focused) without the work in #48461 as it appears we'd need to set the Tooltip at the Button component level, which is currently set to `aria-hidden`. Enabling the tooltip there would likely require #48461 so that we can use the `aria-label`. The solution then might be to augment the `ariaLabel` to include the position status, and we could look at ensuring we deal with keyboard accessibility at the same time as accessibility for screen readers.

What do you think? I don't think there's any particular urgency for this PR, so we could always wait for #48461 if we think displaying the tooltip for users navigating by keyboard is a blocker 🙂

Reply:
Sounds good 👍 I'll see what I can do to help out with that other PR.